.github/workflows/tests_cuda.yml

```diff
@@ -0,0 +1,31 @@
+on: [pull_request]
+
+jobs:
+  build-cuda:
+    container:
+      image: nvcr.io/nvidia/pytorch:20.03-py3
```

Comment:
According to https://docs.nvidia.com/deeplearning/frameworks/support-matrix/index.html this image is based on cuda 10.2. Not sure if the 9.2, 10.1 run in it properly.

Reply:
I took this from monai: https://github.com/Project-MONAI/MONAI/blob/master/.github/workflows/pythonapp.yml#L145
they seems to at least test against 10.1

Reply:
As far as I understand it they also use different images for different pytorch packages.

Reply:
I would say that since we use conda it works well. They use pip for installation which I don't think that resolves  well cuda dependencies

Reply:
@wyli any insight about this ?

Reply:
@wyli other proposals to improve our CI are welcomed

Reply:
Looks like that cudatoolkit contains the cuda runtime as well, so it indeed should be fine.

Reply:
Hi @edgarriba, in case you haven't seen this, monai is actually a bit concerned about this one: https://docs.github.com/en/free-pro-team@latest/actions/hosting-your-own-runners/about-self-hosted-runners#self-hosted-runner-security-with-public-repositories and may migrate the self-hosted pipelines to some other alternatives...

Reply:
@wyli thanks. Good to know

Reply:
@wyli I guess for now a solution for us could be to run this workflow on schedule against master once per day